### ISSUE TITLE
Installation guide

### ISSUE THREAD

**Issue by jola5** [jola5/gtv#1]
Installation instructions. 
* How to add to executable path?
* How to add as custom git command?

**Comment by jola5** [comment]
Use case for system wide installation: 

```
curl -sL https://raw.githubusercontent.com/jola5/gtv/master/install.sh | sudo bash -
```

While `install.sh` being something like: 

```
cd /usr/local/bin
git archive --remote=git@github.com:jola5/gtv.git master:src gtv | tar -x
chmod +x gtv
```

**Comment by jola5** [comment]
Renaming `gtv` to the more tedious `git-tag-version` would provide a custom `git tag-version` out of the box. Probably add aliases for ease of use.

**Comment by jola5** [comment]
`git archive` is not supported by github. What a pity. 

http://www.gilesorr.com/blog/git-archive-github.html

Better idea is probably to build the artifact with travis, generate a true github release artifact and provide that.

**Comment by jola5** [comment]
Fixed in 489067f1ca4121f91e494915a7760482db8829e6
